src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -514,13 +514,15 @@ export default {
 			);
 		}
 
-		// EXP-MARS-143-Jul2022
-		this.shareCardLanguageVersion = this.$route.query?.scle;
-		this.$kvTrackEvent(
-			'Thanks',
-			'EXP-MARS-143-Jul2022',
-			this.shareCardLanguageVersion
-		);
+		if (this.$route.query?.utm_campaign.includes('scle')) {
```

Comment:
If `utm_campaign` can be undefined, you probably want optional chaining before `includes` as well.